web/src/components/tasks/business-formation/review/section/ReviewSubSection.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactElement, ReactNode } from "react";
+
+interface Props {
+  header: string;
+  marginOverride?: string;
+  children?: ReactNode;
+}
+
+export const ReviewSubSection = (props: Props): ReactElement => {
+  return (
+    <>
+      <div className={`flex space-between ${props.marginOverride ?? ""}`}>
+        <div className={"maxw-mobile-lg"}>
+          <h2 className={"h3-styling"}>{props.header}</h2>
```

Comment:
Shouldn't this just be an `h3`? And then you don't need "h3-styling" as a class anyways.

From an accessibility standpoint, the `h` hierarchy should reflect the actual page hierarchy. h3s should be nested inside h2s, and my understanding is that these subsections are nested inside the h2 main sections?

Reply:
I agree. This was carried over from `ReviewSection` (nka `ReviewMainSection`). I should have brought it to you when I saw it then.